src/components/navigation-page/navigation-page.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+export default class NavigationPage extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: null,
+      activeParent: null
+    };
+  }
+
+  componentDidMount() {
+    const { location, parentPage } = this.props;
+
+    this.setState({
+      activeItem: location.pathname,
+      activeParent: parentPage
+    });
+  }
+
+  renderHeader(href, label) {
```

Comment:
I mocked up some suggestions on styling. I think your structure is really good, but we can play with the spacing and color to improve the hierarchy structure between main navigation items and subpages:

![image](https://user-images.githubusercontent.com/2180540/97583086-ea4efd00-19cc-11eb-9b2a-08f9a574278e.png)

Reply:
Thanks for this, Katy! I took your suggestions and run with them a bit: 

1. The main "Tab" section is bubbled now. I didn't love the fully round highlights (to me, they look a bit too much like buttons), so I tried only rounding the left side and I like it! It feels a bit like a "Tab," go figure.
2. Chevron-y icons are in place!
3. I removed a lot of whitespace, which I think makes the hierarchy a bit clearer at a glance.

<img width="302" alt="image" src="https://user-images.githubusercontent.com/2365503/97769234-61c47f80-1ae6-11eb-9476-435a5a39be15.png">